GliaWidgets/ViewController/Call/CallViewController.swift

```diff
@@ -8,11 +8,6 @@ class CallViewController: EngagementViewController, MediaUpgradePresenter {
         super.init(viewModel: viewModel, viewFactory: viewFactory)
     }
 
-    @available(*, unavailable)
-    required init?(coder: NSCoder) {
-        fatalError("init(coder:) has not been implemented")
-    }
-
```

Comment:
Hmm, it was vice versa for me. Xcode 12.2, which Bitrise was configured with, refused to compile with these. When I switched it to Xcode 12.5, it was exactly like on my machine - no errors, no warnings. Curious 😄